spec/features/users/key_pair_generation_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'Generate key pair on Sign in' do
+  let(:percentage) { 0 }
+
+  before do
+    allow(IdentityConfig.store).to receive(:key_pair_generation_percent).and_return(percentage)
+    AbTests.reload_ab_test_initializer!
+  end
+
+  after do
+    allow(IdentityConfig.store).to receive(:key_pair_generation_percent).and_call_original
+    AbTests.reload_ab_test_initializer!
+  end
```

Comment:
I see that we moved `reload_ab_test_initializer!` to the class... I am moderately against this behavior.

In the PR to clean that up, I added `FakeAbTestBucket` which can be "pre-programmed" to return certain values: https://github.com/18F/identity-idp/blob/61b2a4f75eb23292b269fc913e0f9f8a1880d3f1/spec/services/idv/steps/document_capture_step_spec.rb#L67-L70

We should be able to use it in most circumstances, even this one. I'll make a PR to clean up

Reply:
see https://github.com/18F/identity-idp/pull/7093